yml/imsim.yaml

```diff
@@ -8812,7 +8885,7 @@ tables:
   - name: llcra
     '@id': '#CcdVisit.llcra'
     datatype: double
-    description: RA of lower left corner.
+    description: Right Ascension of lower left corner.
```

Comment:
No need to change this now, but I just realized how ambiguous "lower left corner" is (on the sky? which way is up on the camera?). Someday this should probably be "minimum y, minimum x" or some other way to express pixel coordinates.

Reply:
It may or may not help if we linked to the cameraGeom docs in these kinds of places, since we do have a self-consistent camera geometry definition.

https://pipelines.lsst.io/modules/lsst.afw.cameraGeom/cameraGeom.html#camera-coordinate-systems